server/rpc.go

```diff
@@ -242,6 +245,9 @@ func getOrchestrator(context context.Context, orch Orchestrator, req *net.Orches
 	if os != nil && os.IsExternal() {
 		tr.Storage = []*net.OSInfo{os.GetInfo()}
 	}
+
+	tr.TicketParams = orch.TicketParams(addr)
```

Comment:
Since TicketParams is unconditionally assigned, can we add this where `tr` is defined?